client-react/package.json

```diff
@@ -8,6 +8,7 @@
     "upload-coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "test": "rimraf ./.nyc_output ./coverage && cross-env nyc --reporter=lcov --reporter=text mocha --require config/test/mocha-setup.js --recursive src/**/*.spec.js --require ignore-styles",
     "start": "cross-env NODE_ENV=development showroom-scan src && babel-node --presets es2015,stage-0 www/index",
+    "start-wmd": "cross-env NODE_ENV=development showroom-scan src && babel-node --presets es2015,stage-0 win/index-windows",
```

Comment:
I'll add it for this PR. But @estambakio-sc not forget find a way to pass env variables on MS Windows and remove this goal.